backend/src/test/app.test.ts

```diff
@@ -191,11 +193,12 @@ describe("Email Roll", () => {
             .send({ Election: testInputs.EmailRollElection, VoterIDList: testInputs.EmailRoll })
         // console.log(response.body)
         expect(response.statusCode).toBe(200)
-        // id = response.body.election.election_id
+        const responseObject = JSON.parse(response.body)
+        ID = responseObject.election.election_id
     })
     test("Get voter auth, is authorized and hasn't voted", async () => {
         const response = await request(app)
-            .post(`/API/Election/7/ballot`)
+            .post(`/API/Election/${ID}/ballot`)
```

Comment:
If I'm understanding this correctly, it looks like ID could be either 0 if "Create election, responds 200" wasn't previously called, or > 0 if "Create election, responds 200" was previously called. Could this create issues? 

Maybe "Create election, responds 200" should be part of beforeAll?